scripts/qemu_launcher.sh

```diff
@@ -0,0 +1,158 @@
+#!/bin/bash
+
+# Copyright (C) 2017-2018 Luxoft
```

Comment:
To me this is OK. @jeremiah could you please verify?

Reply:
Fine with me. :-) 

As a side note however, copyright lasts for 70+ years past the life of the author. So we can just put one year in there, we don't have to put a range. This is trivial and pedantic of me to me to even mention, but there it is.